qiita_db/test/test_commands.py

```diff
@@ -134,8 +136,8 @@ def test_load_artifact_from_cmd_biom(self):
             with open(fp, 'w') as f:
                 f.write("\n")
         obs = qdb.commands.load_artifact_from_cmd(
-            fps, ftypes, 'BIOM', parents=[3], processing_command_id=3,
-            processing_parameters_id=1)
+            fps, ftypes, 'BIOM', parents=[3], dflt_params_id=10,
+            required_params='{"input_data": 3}')
```

Comment:
Yeah, that's emulating what would happen when a user loads an artifact from the CLI :)